backend/src/common/config/config.package.ts

```diff
@@ -59,6 +61,15 @@ class Config implements IConfig {
                 ACCESS_TOKEN_EXPIRES_IN: '24h',
                 REFRESH_TOKEN_EXPIRES_IN: '30d',
```

Comment:
Make sure that the cookie's lifetime expires at the same time as the refresh token's lifetime